src/lolex-src.js

```diff
@@ -738,7 +738,9 @@ function withGlobal(_global) {
             Object
                 .getOwnPropertyNames(_global.Performance.prototype)
                 .forEach(function (name) {
-                    clock.performance[name] = proto[name];
+                    if (Object.getOwnPropertyDescriptor(proto, name).writable) {
+                        clock.performance[name] = proto[name];
```

Comment:
Changed this in #203 to just overwrite with no-ops. Not sure of any negative implications. Input?